0162-semantic-convention-resource-stability.md

```diff
@@ -0,0 +1,110 @@
+# Resource Semantic Convention Stabiltiy Requirements
+
+Outline what changes are "safe" to perform for resource semantic conventions
+and which require major version bumps.
```

Comment:
I am a bit confused as to why we need this document.

I believe that now that starting from 1.4.0 we have Telemetry Schemas, the Schemas defines what changes are safe. If the change is representable as version change in the Schema then it is safe, everything else is unsafe.